### PR TITLE
IBX-7909: `Content item` changed to `content item` in Developer documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://issues.ibexa.co/browse/IBX-7909)
| Versions      | master

`Content item` changed to `content item` in Developer documentation.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date

### CODE REVIEW

File: code_samples/api/public_php_api/src/Command/UpdateContentCommand.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $contentDraft = $this->contentService->updateContent($contentDraft->versionInfo, $contentUpdateStruct);
         $this->contentService->publishVersion($contentDraft->versionInfo);
 
-        $output->writeln('Content item ' . $contentId . ' updated with new name: ' . $newName);
+        $output->writeln('content item ' . $contentId . ' updated with new name: ' . $newName);
```
Comment (adriendupuis) [inline code_samples/api/public_php_api/src/Command/UpdateContentCommand.php:57]: I wouldn't change this one as this the first world of a sentence

```suggestion
        $output->writeln('Content item ' . $contentId . ' updated with new name: ' . $newName);
```

File: docs/content_management/locations.md [hunk 2]
```diff
@@ -89,33 +89,33 @@ Location visibility allows you to control which parts of the content tree are av
...
 
-All Locations and Content items are visible by default.
+All Locations and content items are visible by default.
 If a Location is made invisible manually, its status is set to Hidden.
 All Locations under it will change status to Hidden by superior.
-A Content item is Hidden by superior only in Locations in which it has a parent Location with the Hidden status.
+A content item is Hidden by superior only in Locations in which it has a parent Location with the Hidden status.
 
-In the following example, the **Content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
+In the following example, the **content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
```
Comment (adriendupuis) [inline docs/content_management/locations.md:118]: I wouldn't downcase this one as this is the label in the schema/illustration [locations_visibility.png](https://github.com/ibexa/documentation-developer/blob/IBX-7909/docs/content_management/img/locations_visibility.png)
```suggestion
In the following example, the **Content item 1** is Hidden by superior in the **Location A** while still visible in the **Location B**.
```

File: docs/templating/templates/templates.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ Main variables include:
 
 |Variable |Description|
 |------|------|
-|`content`|Content item, containing all Fields and version information (VersionInfo). |
+|`content`|content item, containing all Fields and version information (VersionInfo). |
```
Comment (adriendupuis) [inline docs/templating/templates/templates.md:52]: In this table, every description sentence starts with a capital.
```suggestion
|`content`|Content item, containing all Fields and version information (VersionInfo). |
```

File: docs/templating/twig_function_reference/content_twig_functions.md [hunk 4]
```diff
@@ -55,15 +55,15 @@ You can pass one of the following rendering methods to `ibexa_render()`:
...
 
 The function uses prioritized languages from SiteAccess settings unless you pass another language as `forcedLanguage`.
-If the Content item does not have a translation in the prioritized or passed language,
+If the content item does not have a translation in the prioritized or passed language,
 the function returns the name in the main language.
 
 | Argument | Type | Description |
 |---------------|------|-------------|
-| `content` | `Ibexa\Contracts\Core\Repository\Values\Content\Content`</br>or</br>`Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo` | Content item or its ContentInfo object.|
+| `content` | `Ibexa\Contracts\Core\Repository\Values\Content\Content`</br>or</br>`Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo` | content item or its ContentInfo object.|
```
Comment (adriendupuis) [inline docs/templating/twig_function_reference/content_twig_functions.md:66]: Sentence start like line 94
```suggestion
| `content` | `Ibexa\Contracts\Core\Repository\Values\Content\Content`</br>or</br>`Ibexa\Contracts\Core\Repository\Values\Content\ContentInfo` | Content item or its ContentInfo object.|
```

**Comment by julitafalcondusza** [comment]
> Thanks! As Adrien mentioned, it would be good to regenerate the REST reference to make sure all the changes are included (unless it's done already?)

It's done, should work now :)

**Review by adriendupuis — APPROVED** [review]

**Review by dabrt — APPROVED** [review]

**Pull request merged**